dadjokes-onchain/foundry/src/DadJokes.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract DadJokes {
+    struct Joke {
+        string setup;
+        string punchline;
+        address creator;
+        uint256 classicRewards;
+        uint256 funnyRewards;
+        uint256 groanerRewards;
+        bool isDeleted;
+    }
+
+    Joke[] private jokes;
```

Comment:
Jokes Array is perfectly fine. Is there any merit in doing as a mapping instead so then we're not storing "deleted" jokes?  up to you. Minor comment since this is a toy for video.

Reply:
Thanks for pointing this one out! Updated to be a mapping